.travis.yml

```diff
@@ -22,7 +22,7 @@ jobs:
       rust: stable
       cache: cargo
       script:
-        - cargo test --locked --no-default-features
+        - (cd proxy && exec cargo test --locked --no-default-features)
```

Comment:
In proxy/Dockerfile we have `cargo test -p conduit-proxy --frozen --no-default-features`. Unless there's a specific reason why we can't do so, I think it's good to be consistent. Which is better?

Reply:
`cargo test -p conduit-proxy --frozen --no-default-features` will not correctly pass the `--no-default-features` flag; the flaky tests will still be run. I can change the Dockerfile to `cd` into the `proxy` dir as well; I forgot about that.

Reply:
why is `exec` necessary?

Reply:
@olix0r so that we don't create a new process --- [StackOverflow said to](https://stackoverflow.com/a/786515) and it seemed like good hygiene?

Reply:
prefer to omit exec -- it's not used consistently and doesn't solve a problem we have